File: src/main/java/com/team573/gongguri/domain/member/entity/Univ.java
package com.team573.gongguri.domain.member.entity;

import com.team573.gongguri.global.entity.BaseEntity;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor
@Table(name = "univ")
public class Univ extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long univId;

    @Column(nullable = false)
    private String univName;

    public Univ(String univName) {
        this.univName = univName;
    }
}

File: src/main/java/com/team573/gongguri/domain/member/entity/Univ.kt
package com.team573.gongguri.domain.member.entity

import com.team573.gongguri.global.entity.BaseEntity
import jakarta.persistence.*
import lombok.Getter
import lombok.NoArgsConstructor

@Entity
@Table(name = "univ")
class Univ(

    @Column(nullable = false)
    val univName: String

) : BaseEntity() {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val univId: Long? = null

}